application/cms/filters.py

```diff
@@ -74,3 +74,7 @@ def models_to_dicts(items):
 # (e.g. bullet points) copy-and-pasteable from the exported version back into the form fields
 def html_line_breaks(string):
     return Markup(bleach.clean(string).replace("\n", "<br />") if string else "")
+
+
+def url_with_line_breaks(string):
+    return string.replace("/", "/​")  # Adds a zero-width space after the /
```

Comment:
Can we include the zero-width space using its unicode code point, so that we can see it more clearly?

```suggestion
    return string.replace("/", "/​\u200B")  # Adds a zero-width space after the /
```

Reply:
I tried that, but it seemed to get escaped?